includes/options/class-amp-options-manager.php

```diff
@@ -32,14 +32,14 @@ class AMP_Options_Manager {
 		Option::SUPPORTED_TEMPLATES     => [ 'is_singular' ],
 		Option::VERSION                 => AMP__VERSION,
 		Option::READER_THEME            => AMP_Reader_Themes::DEFAULT_READER_THEME,
-		Option::WIZARD_COMPLETED        => false,
+		Option::PLUGIN_CONFIGURED       => false,
+		Option::MOBILE_REDIRECT         => false,
 	];
 
 	/**
 	 * Sets up hooks.
 	 */
 	public static function init() {
-		add_action( 'admin_notices', [ __CLASS__, 'render_welcome_notice' ] );
 		add_action( 'admin_notices', [ __CLASS__, 'render_php_css_parser_conflict_notice' ] );
 		add_action( 'admin_notices', [ __CLASS__, 'insecure_connection_notice' ] );
 	}
```

Comment:
Tested these notices with the new settings screen, just to make sure:

<img width="1511" alt="Screen Shot 2020-07-07 at 10 34 53 PM" src="https://user-images.githubusercontent.com/9020968/86872537-384c9780-c0a2-11ea-9b72-4b7fcd3104a1.png">